README.md

```diff
@@ -61,5 +61,7 @@ npm run build
 <td align="center"><a href="https://github.com/mindninjaX"><img src="https://avatars.githubusercontent.com/mindninjaX?s=100" width="100px;" alt=""  style="border-radius: 50px" /><br /><sub><b>Rishabh Singh</b></sub></a><br /><p>UI/UX Designer</p></td>
 <td align="center"><a href="https://github.com/AdityaTeltia"><img src="https://avatars.githubusercontent.com/AdityaTeltia?s=100" width="100px;" alt=""  style="border-radius: 50px" /><br /><sub><b>Aditya Teltia</b></sub></a><br /><p>Contributor</p></td>
 
+<td align="center"><a href="https://github.com/mdPial"><img src="https://avatars.githubusercontent.com/mdPial?s=100" width="100px;" alt=""  style="border-radius: 50px" /><br /><sub><b>Md. Pial Ahamed</b></sub></a><br /><p>Contributor</p></td>

```

Comment:
I am handling this to you, remove all the `style="border-radius: 50px"` since it will not work in markdown readme rendering.

Remove it from the above `td` tags too, and put your data just after the previous data without line-break.

Reply:
Done. Please check.